Stylesheets/components/badge.css

```diff
@@ -15,59 +20,57 @@
   position: relative;
 }
 
-.ml {
+.mail {
   left: 4rem;
   bottom: 6rem;
 }
 
 .cart {
-  left: 12.5rem;
-  bottom: 8rem;
+  left: 11rem;
+  bottom: 7.8rem;
 }
 .phone {
-  left: 19.7rem;
+  left: 18rem;
   bottom: 10rem;
 }
 .insta {
-  left: 27.3rem;
+  margin-left: 4rem;
   bottom: 12rem;
 }
 
 .hero-img {
   height: 100px;
   width: auto;
   border-radius: 50%;
-  margin: 2rem;
 }
 
 .social {
-  margin: 2rem;
-  height: 30px;
-  width: 30px;
-  border: 2px solid white;
+  height: 2rem;
+  width: 2rem;
+  border: 0.2rem solid white;
   border-radius: 50%;
-  position: relative;
+  position: absolute;
 }
 
-.ol {
+.online {
   background-color: var(--badge-online);
   left: 4rem;
-  bottom: 6rem;
+  top: 4.5rem;
 }
 
-.id {
+.idle {
   background-color: var(--badge-idle);
   left: 14.5rem;
-  bottom: 10rem;
+  top: 4.5rem;
 }
 
-.off {
+.offline {
   background-color: var(--badge-dnd);
   left: 25rem;
-  bottom: 13.8rem;
+  top: 4.5rem;
 }
 
 .hero {
-  position: relative;
-  bottom: 7rem;
+  position: absolute;
```

Comment:
hero is parent div, you can't position it absolute.

Learn more about absolute, relative position

Reply:
yep , i have changed it